src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js

```diff
@@ -44,18 +61,14 @@ const MappingProfilesContainer = ({
             sort: new URLSearchParams(window.location.search).get('sort') || '',
             query: new URLSearchParams(window.location.search).get('query') || '',
           },
-          mappingProfiles: {
-            records: tempData.profiles,
-            hasLoaded: true,
-            isPending: false,
-            other: { totalRecords: 4 },
-          },
+          ...resources,
         }}
         parentMutator={{
+          ...mutator,
           resultCount: { replace: noop },
           resultOffset: { replace: noop },
         }}
-        formatter={getMappingProfilesItemFormatter({ format: record => record.format })}
+        formatter={getMappingProfilesItemFormatter({ format: ({ outputFormat }) => outputFormat ? outputFormat : <NoValue /> })}
```

Comment:
According to PO comment no dash is needed when there is nothing to render.

```suggestion
        formatter={getMappingProfilesItemFormatter({ format: ({ outputFormat }) => outputFormat ? outputFormat : null })}
```